Shared/mods/deathmatch/logic/Utils.h

```diff
@@ -246,6 +246,21 @@ inline float GetSmallestWrapUnsigned(float fValue, float fHigh)
 
 void RotateVector(CVector& vecLine, const CVector& vecRotation);
 
+inline void ConvertEulersToQuaternion(const CVector& vecFrom, CVector4D vecTo)
+{
+    double cy = cos(vecFrom.fZ * 0.5);
```

Comment:
Trigonometric functions accept radians. But from what I can see `vecFrom` contains degrees.

Reply:
Oh, yes. This is a mess